package.json

```diff
@@ -14,50 +14,62 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.17.2",
-    "ember-cli-htmlbars": "^4.2.2"
+    "ember-cli-babel": "^7.23.0",
+    "ember-cli-htmlbars": "^5.3.1"
   },
   "peerDependencies": {
     "@lblod/ember-vo-webuniversum": "^0.22.1",
     "@lblod/marawa": "^0.6.0",
```

Comment:
perhaps bump marawa to the latest version

Reply:
I've bumped it in rdfa-editor without issue, but some testing doesn't hurt :)

Reply:
I did and tested but not sure that I understand this correctly, it says that the peer should have at least marawa 0.6, shouldn't we try to support a version of marawa as low as possible for compatibility